src/lookup.rs

```diff
@@ -1,13 +1,14 @@
-use p3_air::{Air, AirBuilder, AirBuilderWithPublicValues, BaseAir, BaseAirWithPublicValues};
+use p3_air::{Air, BaseAir, BaseAirWithPublicValues, ExtensionBuilder};
 use p3_field::{Algebra, Field, PrimeCharacteristicRing, batch_multiplicative_inverse};
 use p3_matrix::{Matrix, dense::RowMajorMatrix};
 
 use crate::{
     builder::{PreprocessedBuilder, TwoStagedBuilder, symbolic::SymbolicExpression},
-    system::MIN_IO_SIZE,
-    types::Val,
+    types::{ExtVal, Val},
 };
 
+pub const LOOKUP_PUBLIC_SIZE: usize = 4;
```

Comment:
Please add a docstring saying what these public values are.
This is what we had for `MIN_IO_SIZE`:
```
/// Each circuit is required to have at least 4 arguments. Namely, the lookup challenge,
/// fingerprint challenge, current accumulator and next accumulator
```